solidity/contracts/deposit/DepositFunding.sol

```diff
@@ -97,13 +97,13 @@ library DepositFunding {
         // refund the deposit owner the cost to create a new Deposit at the time the Deposit was opened.
         uint256 _seized = _d.seizeSignerBonds();
 
-        /* solium-disable-next-line security/no-send */
-        _d.depositOwner().send(_d.keepSetupFee);
-        _d.pushFundsToKeepGroup(_seized.sub(_d.keepSetupFee));
-
         _d.setFailedSetup();
         _d.logSetupFailed();
 
+        /* solium-disable-next-line security/no-send */
+        _d.depositOwner().call.value(_d.keepSetupFee)("");
+        _d.pushFundsToKeepGroup(_seized.sub(_d.keepSetupFee));
+
```

Comment:
We may want to leave a note as to why we're doing the state change in `fundingTeardown` after we do the above transfers.

Reply:
there's no reason here. We can do it after as long as we ensure state change comes first.